packet/flatcar-linux/kubernetes/outputs.tf

```diff
@@ -5,3 +5,34 @@ output "kubeconfig-admin" {
 output "kubeconfig" {
   value = module.bootkube.kubeconfig-kubelet
 }
+
+output "dns_entries" {
+  value = concat(
+    # etcd
+    [
+      for device in packet_device.controllers:
```

Comment:
You could just do `for index, device in packet_device.controllers` IIRC. Then there is no need to call a function :wink:

Reply:
Thanks!